### PR TITLE
build(ci): add on-push restarts

### PR DESCRIPTION
**Pull request by nandor-magyar**
Invoke `kubectl rollout restart` for components used / built here.

### CODE REVIEW

File: .github/workflows/full.yml [hunk 1]
```diff
@@ -781,3 +782,31 @@ jobs:
           cosign sign --yes --key cosign.key $(docker inspect --format='{{index .RepoDigests 0}}' ${DOCKERHUB_REGISTRY}/kratos:${{ needs.gather_changes.outputs.tag }} )
         env:
           COSIGN_PASSWORD: ${{ secrets.COSIGN_PASSWORD }}
+  crux_test_deploy:
```
Comment (polaroi8d) [inline .github/workflows/full.yml:785]: Not proper naming: `crux_test_deploy`

File: .github/workflows/full.yml [hunk 2]
```diff
@@ -781,3 +782,31 @@ jobs:
...
+    runs-on: ubuntu-22.04
+    environment: staging
+    # needs: [crux_push]
```
Comment (polaroi8d) [inline .github/workflows/full.yml:788]: Unnecessary comment.

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: .github/workflows/full.yml (hunk 2 above)
Comment (nandor-magyar) [reply .github/workflows/full.yml:788]: It will be uncommented if ready.

Comment (polaroi8d) [reply .github/workflows/full.yml:788]: Okay, it was not draft :')

**Comment by nandor-magyar** [comment]
Set to be done later, AD and GitHub is not willing to cooperate using their docs, contacting support.

**Pull request closed without merging**